src/shared/util/spawn.js

```diff
@@ -13,14 +13,18 @@ specific language governing permissions and limitations under the License.
 import cp from 'child_process';
 
 import colors from 'colour';
+import { IS_PACKAGED_BUILD } from '../build-info';
 
 export const spawn = (command, main, args, { logger }, options = {}) => new Promise((resolve, reject) => {
   logger.log('Spawning',
     colors.cyan(command),
     colors.blue(main),
     colors.yellow(args.join(' ')));
 
-  const child = cp.spawn(command, [main, ...args], { stdio: 'inherit', ...options });
+  const stdio = process.platform === 'win32' && IS_PACKAGED_BUILD
+    ? 'ignore'
+    : 'inherit';
```

Comment:
This isn't that crucial of an architectural problem at all, but it might make sense to think about the purposes of this function.

Do all spawn processes need the `.cmd` extension on Windows? If so, then we should abstract that away in this function. Otherwise, maybe a different solution would make sense.

Similarly, is the `stdio` problem happening everywhere in node/io/electron on Windows, or is it electron-specific? If the former, it might be better to not abstract it away.

What trips me the most though is the `IS_PACKAGED_BUILD` check. It's very surprising to me that this problem doesn't manifest itself when running via commandline.